AnnotationsAndModifiers.md

```diff
@@ -0,0 +1,33 @@
+# Annotations and Modifiers
+
+## Annotations
+
+Method/variable annotations should go on the _same_ line as the rest of the definition. i.e.
+
+```swift
+// good: On the same line
+@objc func doThing() { }
+
+// bad: Not on the same line
+@objc
+func doThing() { }
+```
+
+There is an exception for when you are renaming for Objective-C. In that case, the annotation _may_ go on the preceeding line. e.g.
+
+```swift
+@objc(XYThing)
+enum Thing: Int {
+...
+}
+```
+
+Regarding ordering, the rules defined by SwiftLint should be followed.
```

Comment:
What does the ordering refer to? The ordering of the annotations? If so, could we add a link to a documentation here instead of saying SwiftLint?